exporter/splunkhecexporter/config.go

```diff
@@ -68,6 +69,10 @@ type Config struct {
 	exporterhelper.QueueSettings `mapstructure:"sending_queue"`
 	configretry.BackOffConfig    `mapstructure:"retry_on_failure"`
 
+	// Experimental: This configuration is at the early stage of development and may change without backward compatibility
+	// until https://github.com/open-telemetry/opentelemetry-collector/issues/8122 is resolved.
+	BatcherConfig exporterbatcher.Config `mapstructure:"batcher"`
```

Comment:
Would it make sense to include an example in the README of how to use the batcher config? I realize it's changing so we may end up getting a stale config, so I'll leave it up to you 👍

Reply:
yeah since it is experimental maybe we can leave it out for now. I think at some point there should be a centralized reference in exporter helper with examples. Since every exporter(that are interested in batching) can point to that.